crun.1.md

```diff
@@ -294,6 +294,20 @@ Path to the file containing the resources to update.
 
 # Extensions to OCI
 
+## `run.oci.seccomp.receiver=PATH`
+
+If the annotation `run.oci.seccomp.receiver=PATH` is specified, the
+seccomp listener is sent to the UNIX socket listening on the specified
+path.  It can also set with the `RUN_OCI_SECCOMP_RECEIVER` environment variable.
+It is an experimental feature, and the annotation will be removed once
+it is supported in the OCI runtime specs.  It must be an absolute path.
+
+## `run.oci.seccomp.plugins=PATH`
+
+If the annotation `run.oci.seccomp.plugins=PLUGIN1[:PLUGIN2]...` is specified, the
+seccomp listener fd is handled through the specified plugins.  The
+plugin must either be an absolute path or a filename.
```

Comment:
What does "filename" mean?

Reply:
I've changed it to: `a file name without any slash`

Reply:
In LD_LIBRARY_PATH?

Reply:
it is more complicated than that.  I've added a pointer to the ld.so(8) man page that describes how the lookup is performed.